pytorch_lightning/trainer/trainer.py

```diff
@@ -138,6 +138,7 @@ def __init__(
         move_metrics_to_cpu: bool = False,
         enable_pl_optimizer: bool = None,  # todo: remove in v1.3
         multiple_trainloader_mode: str = 'max_size_cycle',
+        use_swa: bool = False
```

Comment:
I would prefer to have it in kwargs, the adding beta feature to stable Trainer api

Reply:
I agree. Can SWA also be used without that flag? I.e. by adding a callback? 
Then INO we shouldn't add a flag for this while this is in beta.

Reply:
I agree. Making a vote !

Reply:
Will and Ananya agued it is well known techniques and it can be added.